modules/core/src/main/resources/scalafix-migrations.conf

```
@@ -89,6 +89,12 @@ migrations = [
     rewriteRules: ["https://gist.githubusercontent.com/fthomas/ea14d58e1b445d8038a5bc22acf751e8/raw/e1180c91dab0b36d8f9ec37d413ef9507bee67ff/log4cats-1.2.0.scala"],
     authors: ["Frank Thomas <frank@timepit.eu>"]
   },
+  {
+    groupId: "org.gnieh,
+    artifactIds: ["fs2-data-json.*"],
+    newVersion: "1.8.0",
+    rewriteRules: ["github:gnieh/fs2-data/json-parse?sha=v1.8.0"]
```

```suggestion
    rewriteRules: ["github:gnieh/fs2-data/json-parse?sha=v1.8.0"],
    doc: "https://github.com/scala-steward-org/scala-steward/pull/3123"
```

What about adding this PR as documentation for the scalafix migration ?